controller.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"flag"
+	"os"
+
+	// Import all Kubernetes client auth plugins (e.g. Azure, GCP, OIDC, etc.)
+	// to ensure that exec-entrypoint and run can make use of them.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/spf13/cobra"
+
+	orgv1 "github.com/appuio/control-api/apis/organization/v1"
+	controlv1 "github.com/appuio/control-api/apis/v1"
+
+	"github.com/appuio/control-api/controllers"
+	//+kubebuilder:scaffold:imports
+)
+
+// ControllerCommand creates a new command allowing to start the controller
+func ControllerCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "controller",
+	}
+
+	zapfs := flag.NewFlagSet("zap", flag.ExitOnError)
+	opts := zap.Options{}
+	opts.BindFlags(flag.CommandLine)
+	cmd.Flags().AddGoFlagSet(zapfs)
+
+	metricsAddr := cmd.Flags().String("metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	enableLeaderElection := cmd.Flags().Bool("leader-elect", false,
+		"Enable leader election for controller manager. "+
+			"Enabling this will ensure there is only one active controller manager.")
+	probeAddr := cmd.Flags().String("health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	usernamePrefix := cmd.Flags().String("username-prefix", "", "Prefix prepended to username claims. Usually the same as \"--oidc-username-prefix\" of the Kubernetes API server")
+
+	cmd.Run = func(*cobra.Command, []string) {
+		scheme := runtime.NewScheme()
+		setupLog := ctrl.Log.WithName("setup")
+
+		utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+		utilruntime.Must(orgv1.AddToScheme(scheme))
```

Comment:
Technically you don't need `orgv1` right? (But doesn't hurt either so :shrug:)

Reply:
I don't need it. It was more of a precaution if we need it for the next feature. I don't mind removing or leaving it in. It's a thing you won't see in go tests though. You'll see it on first deploy 😅

Reply:
Fair enough. IMO we can leave it